INSTALL.md

```diff
@@ -216,14 +216,16 @@ step 3.
 **NOTE**: if linking against a non-system GMP, pass `GMP_PREFIX=<path/to/gmp>`
 to the `./configure` step.
 
-## Enabling and linking to Intel HEXL
+## Enabling and linking to Intel® HEXL
+**NOTE:** HEXL is only supported on the processors with AVX512DQ and AVX512-IFMA
```

Comment:
This is not strictly true. It is *recommended* to have AVX512 support but HEXL does work without it.

Reply:
I can change to HEXL **_acceleration_** is only supported ... with AVX512.

Without AVX512 it uses the reference implementation in C++ with no acceleration. in fact slows down when compared with NTL.
if we don't say anything, we would be misleading people.

Reply:
Plus, this is a statement about HElib with HEXL acceleration, and not about HEXL.